src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        Random value = new Random();
+        final int limit = 100;
+        int num = value.nextInt(limit);
+        return new Ball(color.getRandomColor().toString(), num);
+    }
+}
```

Comment:
if you want to create a constant limit = 100 you should define it like this
```suggestion
public class Lottery {
        private static final int LIMIT = 100;

    public Ball getRandomBall() {
        ColorSupplier color = new ColorSupplier();
        Random value = new Random();
        int num = value.nextInt(limit);
        return new Ball(color.getRandomColor().toString(), num);
    }
}
```